packages/assets-controllers/package.json

```diff
@@ -88,7 +88,7 @@
     "typescript": "~4.9.5"
   },
   "peerDependencies": {
-    "@metamask/accounts-controller": "^17.0.0",
+    "@metamask/accounts-controller": "^17.1.0",
```

Comment:
Do we need to bump this one? This may have been bumped automatically by accident.

``` suggestion
    "@metamask/accounts-controller": "^17.0.0",

Reply:
Yes, the linter would complain without it